lms/services/vitalsource/service.py

```diff
@@ -11,7 +11,7 @@
 class VitalSourceService:
     """A high-level interface for dealing with VitalSource."""
 
-    H_SKU = "HYPOTHESISLMSAPP"
+    H_SKUS = ["HYPOTHESISLMSAPP", "HYPOTHESISLMSAPPR180"]
```

Comment:
This could become some sort of configuration but 

- env vars are currently a pain to update keep in sync between envs.
- these are not install specific so a setting at that level does not seem adequate either.

I don't think we'll see a proliferation of these, let's evaluate alternatives if we ever do.

Reply:
It seems sensible for now. We can think on a different approach if we end up having to change licenses on a weekly bases.